src/test/java/step2/LottoTest.java

```diff
@@ -0,0 +1,26 @@
+package step2;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import step2.domain.Lotto;
+import step2.domain.LottoNumbers;
+
+public class LottoTest {
+    @Test
+    @DisplayName("로또 번호 비교 로직 테스트")
+    public void compareTest() {
+        LottoNumbers source = new LottoNumbers(
+            List.of(1, 2, 3, 4, 5, 6)
+        );
+        LottoNumbers target = new LottoNumbers(
+            List.of(1, 2, 3, 4, 5, 7)
+        );
+        Lotto sourceLotto = new Lotto(source);
+        assertEquals(5, sourceLotto.getMatchCount(target));
```

Comment:
```suggestion
        assertEquals(5, sourceLotto.matchCount(target));
```

`get-`패턴을 안붙이는것에 대해서어떻게 생각하시나요?